aten/src/ATen/native/EmbeddingBag.cpp

```diff
@@ -248,9 +366,9 @@ _embedding_bag_cpu(const Tensor &weight, const Tensor &indices,
       if (per_sample_weights.defined()) {
         AT_ASSERT(mode == MODE_SUM);
         index_select_scale_add<scalar_t>(
-            indices, offset2bag, per_sample_weights, weight, output);
+            indices, offset2bag, per_sample_weights, weight, output, offsets);
```

Comment:
you can probably save even more compute by skipping make_offset2bag vector in case we specialize here. Otherwise derivatives of offsets twice.

at least - add a TODO here so that the next person knows it

Reply:
Uh, I think i'm gonna do that. make_offset2bag takes a ton of runtime in my tests and if we can get rid of it we can probably squeeze more perf out of this. Thanks for the tip